src/server/index.js

```diff
@@ -143,16 +139,6 @@ if (env === 'development') {
   ]);
 }
 
-if (env === 'production') {
```

Comment:
@adamgraves-choices I'd suggest doing a new commit that just removes this section (L146-L155)